testing/infra/terraform/modules/soaktest_workers/README.md

```diff
@@ -0,0 +1,58 @@
+## Soaktest worker module
+
+This module sets up worker with load generation binary for soaktest configured as a systemd unit along with required monitoring setup.
+
+## Pre-requisites
+
+1. Follow the steps in [this guide](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started).
+2. The project uses [OS login](https://cloud.google.com/compute/docs/oslogin) to ssh into the created worker node, make sure that your google account has the [required privileges](https://cloud.google.com/compute/docs/oslogin/set-up-oslogin#grant-iam-roles)
```

Comment:
Will this work out of the box for our accounts?

Reply:
Yes, since we have editor access on elastic-apm project